CMakeLists.txt

```diff
@@ -149,6 +149,8 @@ if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR
               "$<TARGET_FILE:jbmc>"
               "$<TARGET_FILE:jdiff>"
               "$<TARGET_FILE:smt2_solver>"
+              "$<TARGET_FILE:symtab2gb>"
```

Comment:
FWIW all this "$<TARGET_FILE:` stuff in here isn't _invalid_ per se, but it also doesn't help. Just listing the target names here would work just as well and look less confusing.

Reply:
@hannes-steffenhagen-diffblue Would it be worth doing a cleanup across the code base? Would you mind driving this as my CMake expertise is, well, not really existent?